app/models/barbeque/sns_subscription.rb

```diff
@@ -0,0 +1,19 @@
+module Barbeque
+  class SNSSubscription < ApplicationRecord
+    belongs_to :job_queue
+    belongs_to :job_definition
+
+    validates :topic_arn,
+      uniqueness: { scope: :job_queue, message: 'should be set with only one queue' },
+      presence: true
+    validates :job_queue, presence: true
+    validates :job_definition, presence: true
+
+    after_update :update_queue_policy!
+
+    private
+    def update_queue_policy!
```

Comment:
In your company, [this is prohibited](https://github.com/cookpad/styleguide/blob/092ff807549c16b7ca6d63e11e5514d540f974ba/ruby.en.md#module-and-class-definitions).

> Put empty lines around the visibility-change methods.